nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/stringsubs/impl/StringSubstitutionParser.java

```diff
@@ -68,6 +69,9 @@ public static StringsubsDefinition parse(InputStream configStream) throws String
             JAXBContext context = JAXBContext.newInstance(StringsubsDefinition.class.getPackage().getName());
             Unmarshaller unmarshaller = context.createUnmarshaller();
             SchemaFactory schemaFactory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
+            schemaFactory.setProperty(XMLConstants.ACCESS_EXTERNAL_DTD, "");
+            schemaFactory.setProperty(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "file");
```

Comment:
Here we need the file protocol, because `stringsubs.xsd` includes `nons_stringsubs.xsd` using `xsd:include`